create_new_module_helper.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Prompt user for project name
+read -p "Enter new module name: " module_name
+
+# Prompt user for application id
+read -p "What kind of module you would like to create: [1] compose [2] desktop [3] KMM: " option
+
+case $option in
+1)
+  plugin_name=compose-module-setup
+  ;;
+2)
+  plugin_name=desktop-application-setup
+  ;;
+3)
+  plugin_name=kmm-module-setup
+  ;;
+*)
+  exit 1
+  ;;
+esac
+
+# checking module_name
+if [ -z "${module_name}" ]; then
+    echo "Empty module name..."
+    exit 1
+fi
+
+# checking option
+if [ -z "${plugin_name}" ]; then
+    echo "Invalid option..."
+    exit 1
+fi
+
+# creating directories
+mkdir "${module_name}"
+mkdir "${module_name}/src"
+mkdir "${module_name}/src/commonMain/"
```

Comment:
I think you don't need those lines.
Only line 40 should be enough to create the directory structure.